src/main/groovy/com/entagen/jenkins/JenkinsApi.groovy

```diff
@@ -224,7 +224,11 @@ class JenkinsApi {
         http.handler.failure = { resp ->
             def msg = "Unexpected failure on $jenkinsServerUrl$path: ${resp.statusLine} ${resp.status}"
             status = resp.statusLine.statusCode
-            throw new Exception(msg)
+            if (status != 500) {
```

Comment:
I am not sure that ignoring all 500 errors is the best course of action here.

What if there is an error creating a job?

It would be nice if we caught the error, printed the message, continued the rest of the operations. And then failed the build at the end.